src/platform/windows.rs

```diff
@@ -235,15 +228,9 @@ pub trait WindowBuilderExtWindows {
 }
 
 impl WindowBuilderExtWindows for WindowBuilder {
-    #[inline]
-    fn with_parent_window(mut self, parent: HWND) -> WindowBuilder {
-        self.platform_specific.parent = Parent::ChildOf(parent);
-        self
-    }
-
     #[inline]
     fn with_owner_window(mut self, parent: HWND) -> WindowBuilder {
-        self.platform_specific.parent = Parent::OwnedBy(parent);
+        self.platform_specific.parent = Some(parent);
```

Comment:
would like to have it renamed to `self.platform_specific.owner` as otherwise we have shared attribute `parent` and platform specific `parent` attribute